webapp/channels/src/components/logged_in/logged_in.test.tsx

```diff
@@ -12,22 +12,55 @@ import BrowserStore from 'stores/browser_store';
 import LoggedIn from 'components/logged_in/logged_in';
 import type {Props} from 'components/logged_in/logged_in';
 
-import {fireEvent, renderWithContext, screen} from 'tests/react_testing_utils';
+import {act, fireEvent, renderWithContext, screen, waitFor} from 'tests/react_testing_utils';
 
 jest.mock('actions/websocket_actions.jsx', () => ({
     initialize: jest.fn(),
     close: jest.fn(),
 }));
 
+jest.mock('utils/timezone', () => ({
+    getBrowserTimezone: jest.fn().mockReturnValue('America/New_York'),
+}));
+
+// Mock the Redirect component since we want to verify it's rendered with specific props
+jest.mock('react-router-dom', () => {
+    const actual = jest.requireActual('react-router-dom');
+    return {
+        ...actual,
+        Redirect: jest.fn(({to}) => (
+            <div
+                data-testid='redirect'
+                data-to={to}
+            />
+        )),
+    };
+});
+
+// Mock the LoadingScreen component
+jest.mock('components/loading_screen', () => {
```

Comment:
Why mock this component? As far as I know, it's simple enough that we should be able to render it without mocking